go/cmd/vttablet/cli/cli.go

```diff
@@ -248,7 +248,10 @@ func createTabletServer(ctx context.Context, env *vtenv.Environment, config *tab
 		addStatusParts(qsc)
 	})
 	servenv.OnClose(qsc.StopService)
-	qsc.InitACL(tableACLConfig, enforceTableACLConfig, tableACLConfigReloadInterval)
+	err := qsc.InitACL(tableACLConfig, tableACLConfigReloadInterval)
+	if err != nil && enforceTableACLConfig {
+		return nil, fmt.Errorf("failed to initialize table acl: %w", err)
+	}
```

Comment:
This is not about reloading the file. We should fail to start when given an invalid config and IMO it's quite dangerous to just log an error and continue on as you'd likely have far more permissive permissions than you intended/expected (and how closely are you watching tablet logs when there's otherwise no visible user impacting issues?).

Reply:
Ah, sorry! We ARE returning an error here. Reading is hard 😆